packages/x-components/src/x-modules/ai/components/ai-overview.vue

```diff
@@ -195,4 +235,22 @@ export default defineComponent({
 .x-ai-overview-expand-btn-icon {
   @apply x-icon;
 }
+.x-ai-overview-suggestion-query-btn {
+  @apply x-button-tight x-mx-16 x-font-bold x-text-gray-900 x-w-fit x-min-h-fit x-flex x-gap-16 x-items-center;
+}
+.x-ai-overview-suggestion-query-btn-icon {
+  @apply x-icon-md;
+}
+.x-ai-overview-suggestions {
+  @apply x-flex x-flex-col x-gap-16;
+}
+.x-ai-overview-suggestion {
+  @apply x-flex x-flex-col x-gap-8;
+}
+.x-ai-overview-suggestion-results {
+  @apply x-flex x-gap-16 x-px-16;
+}
+.x-ai-overview-suggestion-result {
+  @apply x-w-[150px];
```

Comment:
Could we have a way to use the tailwind classes directly in the `template`? Now is really hard to understand what classes are applying to each DOM element